modules/claims_api/app/controllers/claims_api/base_disability_compensation_controller.rb

```diff
@@ -18,8 +18,6 @@ def upload_form_526
       ClaimsApi::ClaimUploader.perform_async(pending_claim.id)
 
       render json: pending_claim, serializer: ClaimsApi::AutoEstablishedClaimSerializer
-    rescue
-      render json: unprocessable_response, status: :unprocessable_entity
```

Comment:
while this was initially removed for debugging purposes, should it stay out or perhaps be updated to be more descriptive?

Reply:
I think the idea of capturing all of the errors was a good one but rushed too quickly in terms of implementation. My recommendation here is instead of removing it do a `rescue => e` and capture the actual error coming in and pass it to the unprocessable_response method to actually give us what the error and message is or might be this way a proper error can be properly processed. Also make sure to send the error using a log to sentry method, we have some examples of that in the controllers.

https://github.com/department-of-veterans-affairs/vets-api/blob/a0df370e2f1bd50367f841c23ff53965be3858fb/modules/claims_api/app/controllers/claims_api/base_disability_compensation_controller.rb#L79-L83